<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDlgCamButtonSet</class>
 <widget class="QWidget" name="QDlgCamButtonSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_userMenu">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>183</y>
     <width>109</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>用户菜单</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_set">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>224</y>
     <width>109</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_exit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>109</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>111</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_left1">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>左侧按键短按</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_left2">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>左侧按键长按</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_right1">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>右侧按键短按</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_right2">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>右侧按键长按</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_center1">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>中心按键短按</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_center2">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>中心按键长按</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>131</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QMyComboBox" name="comboBox_left1">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QMyComboBox" name="comboBox_left2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QMyComboBox" name="comboBox_right1">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QMyComboBox" name="comboBox_right2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>28</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_userMenu">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>184</y>
     <width>101</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(29,59,140);
color: rgb(255, 255, 255);
border:none;</string>
   </property>
   <property name="text">
    <string>用户菜单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>227</y>
     <width>101</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(29,59,140);
color: rgb(255, 255, 255);
border:none</string>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>271</y>
     <width>101</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(29,59,140);
color: rgb(255, 255, 255);
border:none</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMyComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">qmycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
